app/src/models/uncommitted-changes-strategy.ts

```diff
@@ -0,0 +1,5 @@
+export enum UncommittedChangesStrategy {
+  askForConfirmation,
```

Comment:
:art: not worth holding this up, but it'd be great to make these keys `SentenceCase` and also string enums to be consistent with other parts of the app.

Reply:
Tracked in #7279.